Vagrantfile

```diff
@@ -1,16 +1,11 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-ram = 4 #GB
```

Comment:
Actually, maybe we should make this a bit useful instead? set vb.cpus and vb.memory to this X lala etc, and then add environment variables so if you have VAGRANT_RAM=16 you get 16GB ram used for the vagrant machine?